manual/query_builder/insert/README.md

```diff
@@ -87,4 +87,20 @@ insertInto("user").json(bindMarker()).usingTimestamp(bindMarker())
 
 If you call the method multiple times, the last value will be used.
 
-[QueryBuilder]: http://docs.datastax.com/en/drivers/java/4.0/com/datastax/oss/driver/api/query-builder/QueryBuilder.html
+### Time To Live (TTL)
+
+Similar to Timestamps, you can specify data expiration TTLs with a literal value:
```

Comment:
I would harmonize the description between inserts and updates around stg along the lines of "You can generate a `USING TTL` clause that will cause column values to be deleted (marked with a tombstone) after the
specified time (in seconds) has expired. This can be done with a literal: ...".